<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SclangProjectConfigPage</class>
 <widget class="QWidget" name="SclangProjectConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="1">
       <widget class="QCheckBox" name="autoStart">
        <property name="text">
         <string>Start Interpreter With IDE</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Runtime Directory:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ScIDE::PathChooserWidget" name="runtimeDir"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="activeConfigFileLabel">
        <property name="text">
         <string>Current Project:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ide_sclang_override_configuration_file">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Interpreter Options (stored in current project file)</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Include:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item row="0" column="0" rowspan="2">
          <widget class="QListWidget" name="sclang_include_directories"/>
         </item>
         <item row="0" column="2">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolButton" name="sclang_add_include">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="sclang_remove_include">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="excludeLabel">
        <property name="text">
         <string>Exclude:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1" rowspan="3">
          <widget class="QListWidget" name="sclang_exclude_directories"/>
         </item>
         <item row="0" column="3">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QToolButton" name="sclang_add_exclude">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="sclang_remove_exclude">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="sclang_post_inline_warnings">
          <property name="text">
           <string>Post Inline Warnings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="sclang_exclude_default_paths">
          <property name="text">
           <string>Exclude default search paths</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>ide_sclang_override_configuration_file</zorder>
  <zorder>widget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScIDE::PathChooserWidget</class>
   <extends>QLineEdit</extends>
   <header>path_chooser_widget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
